tools/istio-docker.mk

```diff
@@ -164,6 +164,17 @@ docker.app_sidecar_debian_10: $(ISTIO_OUT_LINUX)/server
 	$(RENAME_TEMPLATE)
 	$(DOCKER_RULE)
 
+# Test application bundled with the sidecar (for non-k8s).
+docker.app_sidecar_centos_8: BUILD_ARGS=--build-arg BASE_VERSION=${BASE_VERSION}
+docker.app_sidecar_centos_8: tools/packaging/common/envoy_bootstrap.json
+docker.app_sidecar_centos_8: $(ISTIO_OUT_LINUX)/release/istio-sidecar.rpm
+docker.app_sidecar_centos_8: $(ISTIO_DOCKER)/certs
+docker.app_sidecar_centos_8: pkg/test/echo/docker/echo-start.sh
+docker.app_sidecar_centos_8: $(ISTIO_OUT_LINUX)/client
+docker.app_sidecar_centos_8: $(ISTIO_OUT_LINUX)/server
+docker.app_sidecar_centos_8: pkg/test/echo/docker/Dockerfile.app_sidecar_centos
```

Comment:
Should this be pkg/test/echo/docker/Dockerfile.app_sidecar_centos_8? I think we might need to rename the file because it's using the target directly as the filename.